ftl/core/deck-config.ftl

```diff
@@ -39,8 +39,8 @@ deck-config-limit-interday-bound-by-reviews =
 ## Daily limit tabs: please try to keep these as short as the English version,
 ## as longer text will not fit on small screens.
 
-deck-config-shared-preset = Shared preset
-deck-config-deck-only = Deck only
+deck-config-shared-preset = Preset
+deck-config-deck-only = This deck
 deck-config-today-only = Today only
```

Comment:
I think this could just as well apply to the preset. Maybe "This deck today" would be clearer? Hypothetically, Anki could add "Preset today" in the future.
When that's decided we should pronably adjust the keys as well.

Reply:
Just trying to keep the names as succinct as possible, so they fit well. How about we explain in more detail in the tooltip?